lib/kubernetes-deploy/kubernetes_resource/secret.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class Secret < KubernetesResource
+    TIMEOUT = 10.seconds
```

Comment:
I think this is reasonable, but it is inconsistent with our timeout for other resources like this (e.g. Configmap). We should probably have a constant they all use. 🤔

Reply:
I ~stole~ took inspiration from `PodDisruptionBudget` here 🤷‍♂️ 

Agreed it would be good to pick a standard.